### ISSUE TITLE
Battle Arena Toshinden (USA) needs force recompiler icache on in per game settings

### ISSUE THREAD

**Issue by Kurayami7** [stenzek/duckstation#1790]
With a fresh install of Duck and starting the game from the list for the first time It isn't on by default.  I don't know if it's meant to be on by default and something's broken or if it was never made to be on by default in the first place. But it needs it or the game runs at super speed.

![Capture](https://user-images.githubusercontent.com/77198254/110864794-b5f23280-8316-11eb-90f7-e2dd3d5f3718.PNG)

**Comment by ggrtk** [comment]
In what instances is the game running at "super speed", and how fast relative to real hardware? Assuming that the NTSC-J version runs similar code, I checked this a couple weeks back for issue #501 and wasn't able to pinpoint for any of the CPU execution modes where it was noticeably faster.

**Comment by Kurayami7** [comment]
During gameplay obviously. It runs way too fast until I turn on recompiler icache.

**Comment by stenzek** [comment]
It still runs too fast at times even with icache enabled, hence why I didn't bother. Interpreter with partial icache filling helps a bit more, but even then it still speeds up at times. Might be throttled by GPU timings... poorly written game.

**Comment by ggrtk** [comment]
Just tested again. Seems like certain character and stage combinations cause varying amounts of speedups, and last time I just happened to land on the ones that were close enough to console timings.

**Comment by stenzek** [comment]
Yeah. I'd be quite curious how it compares on different hardware revisions - if it is GPU throttled, it should also speed up on newer consoles.

**Comment by stenzek** [comment]
Added in https://github.com/stenzek/duckstation/commit/75e473897eb69b62b3da4dcd496e18c1480d9de7, but I think this still needs more investigation as it briefly runs too fast in some stages/animations.
